html-block-and-inline/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <div>
+      <p>Price: <s>$19.99</s> $14.99</div>
```

Comment:
Whoops, this has a closing `div` tag, but it should be a closing `p` tag

Reply:
removed, thank you.